integration-test/src/test/java/org/sagebionetworks/ITOpenIDConnectTest.java

```diff
@@ -110,10 +124,18 @@ public void testClientNotVerified() throws Exception {
 		authorizationRequest.setRedirectUri(client.getRedirect_uris().get(0));
 		authorizationRequest.setResponseType(OAuthResponseType.code);
 		authorizationRequest.setScope("openid");
-		authorizationRequest.setClaims(
-				"{\"id_token\":{\"userid\":\"null\",\"email\":null,\"is_certified\":null,\"team\":{\"values\":[\"2\"]}},"+
-				 "\"userinfo\":{\"userid\":\"null\",\"email\":null,\"is_certified\":null,\"team\":{\"values\":[\"2\"]}}}"
-		);
+		Map<String, OIDCClaimsRequestDetails> claimsToRequest = new HashMap<>();
```

Comment:
pass the deleted strings to the POJO parser to make sure it still works.

Reply:
This string in particular causes an issue. The value for the field "userId" is (string) `"null"` and not `null`. The JSON parser can't convert `"null"` to the OIDCClaimsRequestDetails object.

This causes a breaking API change, but the impact of the change may be small.